.gitlab/gitlab-ci.yml

```diff
@@ -6,6 +6,9 @@
 
 # We initialize the nonfree repo, then spawn a sub-pipeline from it
 
+variables:
+  VSIM_TESTS: '["testCluster", "testClusterOffload"]'
```

Comment:
This is the place where all tests must be inserted from now on, right?

Reply:
Yes, exactly